LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2024 Astrolicious
+
```

Comment:
```suggestion

The MIT License applies only to files contained within this distribution that do not explicitly include their own licensing information and are not otherwise licensed under different terms. Files that are governed by other licensing terms are excluded from the provisions of this license.

```